src/components/ui/calendar/components/calendarHeader.component.tsx

```diff
@@ -82,19 +85,45 @@ export class CalendarHeader extends React.Component<CalendarHeaderProps> {
     );
   };
 
+  private renderLeftArrow = (): React.ReactElement => {
+    if(this.props.arrowLeft) {
+      return (
+        <TouchableWithoutFeedback onPress={this.props.onNavigationLeftPress}>
+          {this.props.arrowLeft}
+        </TouchableWithoutFeedback>
+      )
+    }
```

Comment:
Can be omitted by passing arrowLeft to Button's accessory prop.
Then there is no need to wrap this component in Touchable for both us and end user.